packages/xstate-wallet/src/ui/application-workflow.tsx

```diff
@@ -46,6 +48,9 @@ export const ApplicationWorkflow = (props: Props) => {
       {isConfirmCreateChannel(current) && (
         <ConfirmCreateChannel current={getConfirmCreateChannelState(current)} send={props.send} />
       )}
+      {isApplicationChallenging(current) && (
+        <p>app requested a challenging! check ur wallet. u can reject if you want</p>
```

Comment:
UI is `TODO`. I'm assuming it will show a message saying "Challenge transaction sent to your wallet. Please approve or decline.".